server/rockcraft.yaml

```
@@ -0,0 +1,45 @@
+# Dockerfile: https://github.com/kubeflow/model-registry/blob/v0.2.22/Dockerfile
+
+name: kubeflow-model-registry-server
+summary: An image for the server of Kubeflow Model Registry
+description: An image for the server of Kubeflow Model Registry
+version: "v0.2.22"
```

our agreed format for version is to exclude `v`
```suggestion
version: "0.2.22"
```